bindings/rust/src/bindings/generated.rs

```diff
@@ -1,72 +1,73 @@
-/* automatically generated by rust-bindgen 0.61.0 */
-
-include!("./consts.rs");
+/* automatically generated by rust-bindgen 0.64.0 */
 
 use libc::FILE;
 
+pub const FIELD_ELEMENTS_PER_BLOB: usize = 4096;
 pub const BYTES_PER_COMMITMENT: usize = 48;
 pub const BYTES_PER_PROOF: usize = 48;
 pub const BYTES_PER_FIELD_ELEMENT: usize = 32;
-pub const BYTES_PER_BLOB: usize = FIELD_ELEMENTS_PER_BLOB * BYTES_PER_FIELD_ELEMENT;
-type limb_t = u64;
+pub const BYTES_PER_BLOB: usize = 131072;
+pub type limb_t = u64;
 #[repr(C)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
-struct blst_fr {
+pub struct blst_fr {
     l: [limb_t; 4usize],
 }
 #[repr(C)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
-struct blst_fp {
+pub struct blst_fp {
     l: [limb_t; 6usize],
 }
 #[repr(C)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
-struct blst_fp2 {
+pub struct blst_fp2 {
     fp: [blst_fp; 2usize],
 }
 #[repr(C)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
-struct blst_p1 {
+pub struct blst_p1 {
     x: blst_fp,
     y: blst_fp,
     z: blst_fp,
 }
 #[repr(C)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq)]
-struct blst_p2 {
+pub struct blst_p2 {
     x: blst_fp2,
     y: blst_fp2,
     z: blst_fp2,
 }
-type g1_t = blst_p1;
-type g2_t = blst_p2;
-type fr_t = blst_fr;
+pub type g1_t = blst_p1;
+pub type g2_t = blst_p2;
+pub type fr_t = blst_fr;
+#[doc = " Internal representation of a polynomial."]
 #[repr(C)]
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub struct Polynomial {
```

Comment:
Hmm, `Polynomial` is not used by any of the public methods. I guess bindgen caught it because it's in `.h`, but I think it should be moved to `.c`.

(/cc @jtraglia )

Reply:
I noticed this too. I had originally instructed bindgen to blocklist the type but left it here to highlight what you just noticed

Reply:
Ah, yes. The `Polynomial` type should be moved back to the `.c` file. Good idea.

Reply:
Made a PR for this here:
* https://github.com/ethereum/c-kzg-4844/pull/244

Reply:
Just merged #244. Would be great if you could re-run bindgen for this PR please.